### PR TITLE
feat: handle 'Aborting' workflow execution phases

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
Workflow executions now transition from 'running' -> 'aborting' -> 'aborted'. This change handles rendering the 'aborting' phase.

Note: I wasn't sure whether it was worth updating the [workflow status filters](https://github.com/flyteorg/flyteconsole/blob/master/src/components/Executions/filters/statusFilters.ts#L7) since this phase is transitive and ideally short-lived

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
See linked issue below.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1978

## Follow-up issue
_NA_

### CODE REVIEW

**Review by csirius — APPROVED** [review]
LGTM!
Should we reflect this on the actual execution phase ui? Is that part implemented?
